web/rootfs/defaults/config.js

```diff
@@ -40,6 +40,9 @@ var config = {
     // BOSH URL. FIXME: use XEP-0156 to discover it.
     bosh: '//jitsi-meet.example.com/http-bind',
 
+    // Websocket URL
+    websocket: '//jitsi-meet.example.com/xmpp-websocket',
```

Comment:
Did you copy this from the current config.js on meet?

Reply:
`serviceUrl` should be used instead which can be used for xmpp and websocket endpoints

```javascript
// websockets
serviceUrl: 'wss://server.com/xmpp-websocket'

// bosh
serviceUrl: '//server.com/xmpp-websocket'
```